### ISSUE TITLE
Different Clock.tick() method

### ISSUE THREAD

**Issue by RaffaeleFiorillo** [pygame/pygame#3976]
**Description**

Hi, I have been using pygame for quite a few projects and, for games where I want to use a frame-independant approach, I use this trick:

delta = clock.tick(FPS)/1000

delta gives me the time between frames, which I can later use for objects movement. 

I would sugest creating a new method that directly returns the value of delta without having to make the division every frame.

**Comment by illume** [comment]
Hello.

So it seems the next step is to come up with a proposal on how this could be done exactly.


> delta = clock.tick(FPS)/1000

So this gives the results in seconds. Note, for any other readers... Clock.tick already gives the amount of time since the last tick() call. But it returns the results in milliseconds not seconds.

I see two possibilities here:

1) method argument to request seconds

> clock.tick(framerate=30, seconds=True)

2) Clock setting

Then all the methods return seconds instead.

> clock = Clock(seconds=True)
> delta_seconds = clock.tick()

The benefit of the first one is that you don't need to supply or change all the methods with a separate argument.


What do you think?

**Comment by RaffaeleFiorillo** [comment]
I think the first option is good. It also makes clear what the function is doing. In cases where the clock instance is created in a different place (module, file) you wouldn't be confused during debugging.

**Comment by prathamroy** [comment]
I also agree that the first proposal is a better idea because of the aforementioned reasons.

**Comment by krobinson395** [comment]
Hello,
I took the suggestions from the first proposal by @illume and executed it in PR #4162. I'm open for any and all feedback for any revisions I should make if this is still an interest!
